File: app/src/main/java/com/inspiredandroid/linuxcommandbibliotheca/interfaces/OnCraftDatabaseListener.java
package com.inspiredandroid.linuxcommandbibliotheca.interfaces;

/**
 * Created by Simon Schubert
 */
public interface OnCraftDatabaseListener {

    /**
     * Successfully created the database
     */
    void onDatabaseCreateSuccess();

    /**
     * Failed to create database
     */
    void onDatabaseCreateFail();
}

File: app/src/main/java/com/inspiredandroid/linuxcommandbibliotheca/interfaces/OnCraftDatabaseListener.kt
package com.inspiredandroid.linuxcommandbibliotheca.interfaces

/**
 * Created by Simon Schubert
 */
interface OnCraftDatabaseListener {

    /**
     * Successfully created the database
     */
    fun onDatabaseCreateSuccess()

    /**
     * Failed to create database
     */
    fun onDatabaseCreateFail()
}
